cmd/sovereignnode/chainSimulator/chainSimulator.go

```diff
@@ -0,0 +1,98 @@
+package chainSimulator
```

Comment:
All of these files are copy-paste from chain simulator, including tests.
Please stop copy-pasting whole files of code,.
Reuse it, also export tests and re-use them.